[Bug]: Androidでビルドできない

### 概要

Androidのエミュレータ上でテストが行えない

### 再現手順

Androidエミュレータをターゲットとして，flutter run を実行

### 期待される結果

エミュレータ上でアプリが立ち上がる

### 実際の結果

```
Your project is configured with Android NDK 26.3.11579264, but the following plugin(s) depend on a different Android NDK version:
- firebase_core requires Android NDK 27.0.12077973
Fix this issue by using the highest Android NDK version (they are backward compatible).
Add the following to /Users/lightswinner/workspace/soccer_flutter_ver/soccer_app_flutter/android/app/build.gradle.kts:

    android {
        ndkVersion = "27.0.12077973"
        ...
    }

e: file:///Users/lightswinner/workspace/soccer_flutter_ver/soccer_app_flutter/android/app/src/main/kotlin/jp/ac/saitama_u/mech/human/soccerAppFlutter/MainActivity.kt:1:52 Expecting a top level declaration
e: file:///Users/lightswinner/workspace/soccer_flutter_ver/soccer_app_flutter/android/app/src/main/kotlin/jp/ac/saitama_u/mech/human/soccerAppFlutter/MainActivity.kt:1:53 Expecting a top level declaration
e: file:///Users/lightswinner/workspace/soccer_flutter_ver/soccer_app_flutter/android/app/src/main/kotlin/jp/ac/saitama_u/mech/human/soccerAppFlutter/MainActivity.kt:1:54 Expecting a top level declaration
e: file:///Users/lightswinner/workspace/soccer_flutter_ver/soccer_app_flutter/android/app/src/main/kotlin/jp/ac/saitama_u/mech/human/soccerAppFlutter/MainActivity.kt:1:55 Expecting a top level declaration
e: file:///Users/lightswinner/workspace/soccer_flutter_ver/soccer_app_flutter/android/app/src/main/kotlin/jp/ac/saitama_u/mech/human/soccerAppFlutter/MainActivity.kt:1:59 Expecting a top level declaration
e: file:///Users/lightswinner/workspace/soccer_flutter_ver/soccer_app_flutter/android/app/src/main/kotlin/jp/ac/saitama_u/mech/human/soccerAppFlutter/MainActivity.kt:1:60 Expecting a top level declaration
e: file:///Users/lightswinner/workspace/soccer_flutter_ver/soccer_app_flutter/android/app/src/main/kotlin/jp/ac/saitama_u/mech/human/soccerAppFlutter/MainActivity.kt:1:65 Expecting a top level declaration
e: file:///Users/lightswinner/workspace/soccer_flutter_ver/soccer_app_flutter/android/app/src/main/kotlin/jp/ac/saitama_u/mech/human/soccerAppFlutter/MainActivity.kt:1:66 Expecting a top level declaration
e: file:///Users/lightswinner/workspace/soccer_flutter_ver/soccer_app_flutter/android/app/src/main/kotlin/jp/ac/saitama_u/mech/human/soccerAppFlutter/MainActivity.kt:1:82 imports are only allowed in the beginning of file

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:compileDebugKotlin'.
> A failure occurred while executing org.jetbrains.kotlin.compilerRunner.GradleCompilerRunnerWithWorkers$GradleKotlinCompilerWorkAction
   > Compilation error. See log for more details

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

BUILD FAILED in 4s
Error: Gradle task assembleDebug failed with exit code 1
```

### 環境

Macbook Pro M3

`flutter doctor` の出力
```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.29.3, on macOS 15.4.1 24E263 darwin-arm64, locale ja-JP)
[!] Android toolchain - develop for Android devices (Android SDK version 35.0.1)
    ✗ cmdline-tools component is missing
      Run `path/to/sdkmanager --install "cmdline-tools;latest"`
      See https://developer.android.com/studio/command-line for more details.
    ✗ Android license status unknown.
      Run `flutter doctor --android-licenses` to accept the SDK licenses.
      See https://flutter.dev/to/macos-android-setup for more details.
[✓] Xcode - develop for iOS and macOS (Xcode 16.3)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2024.3)
[✓] IntelliJ IDEA Ultimate Edition (version 2024.3.5)
[✓] VS Code (version 1.100.2)
[✓] Connected device (6 available)
    ! Error: Browsing on the local area network for Loading…. Ensure the device is unlocked and attached with a cable or associated with the same local area network
      as this Mac.
      The device must be opted into Developer Mode to connect wirelessly. (code -27)
[✓] Network resources

! Doctor found issues in 1 category.
```

### 参考資料

_No response_

--- a/android/app/build.gradle.kts
+++ b/android/app/build.gradle.kts
@@ -11,7 +11,7 @@ plugins {
 android {
     namespace = "jp.ac.saitama_u.mech.human.soccerAppFlutter"
     compileSdk = flutter.compileSdkVersion
-    ndkVersion = flutter.ndkVersion
+    ndkVersion = "27.0.12077973"
 
     compileOptions {
         sourceCompatibility = JavaVersion.VERSION_11

--- a/android/app/src/main/kotlin/jp/ac/saitama_u/mech/human/soccerAppFlutter/MainActivity.kt
+++ b/android/app/src/main/kotlin/jp/ac/saitama_u/mech/human/soccerAppFlutter/MainActivity.kt
@@ -1,4 +1,4 @@
-package jp.ac.saitama_u.mech.human.soccerAppFlutter-u.mech.human.soccerAppFlutter
+package jp.ac.saitama_u.mech.human.soccerAppFlutter
 
 import io.flutter.embedding.android.FlutterActivity
 

--- a/android/settings.gradle.kts
+++ b/android/settings.gradle.kts
@@ -22,7 +22,7 @@ plugins {
     // START: FlutterFire Configuration
     id("com.google.gms.google-services") version("4.3.15") apply false
     // END: FlutterFire Configuration
-    id("org.jetbrains.kotlin.android") version "1.8.22" apply false
+    id("org.jetbrains.kotlin.android") version "1.9.0" apply false
 }
 
 include(":app")

